examples/aws-native-ts-stepfunctions/index.ts

```diff
@@ -45,11 +45,7 @@ const worldFunction = new awsnative.lambda.Function('worldFunction',
     runtime: "nodejs14.x",
     handler: "index.handler",
     code: {
-      zipFile: `exports.handler = function(event, context, callback){
-                                          var response = event.response;
-                                          const updated = { "response": response + "World!" };
-                                          callback(null, updated);
-                                        };`,
+      zipFile: new pulumi.asset.FileAsset("world_function.js"),
```

Comment:
That's why I'm not a fan of the `zipFile` property... There's no zip in here, right? Should we rename to `code`, as in the classic provider? The motivation is that we are changing the behavior of a property, so we may rename it as well.

Reply:
I'm conflicted on this. I agree that the name is confusing, but I'm also hesitant to change the name at the native provider layer. I would be more open to adding an alias, perhaps. My main concern is that we should make it easy for integrators to build on top of the native layer, so I think we should avoid changing the API in incompatible ways.

Reply:
Okay, that's reasonable. Let's proceed with the current name for now then.

Reply:
Should we consider updating the description field for this attribute to make the confusing semantics clearer?